### PR TITLE
[docs] A page with many elements

### PR DESCRIPTION
**Pull request by ddhodge**
A page that shows many different elements used in the documentation

@netlify /preview/contribute/docs/all-page-elements/

### CODE REVIEW

File: docs/content/preview/contribute/docs/all-page-elements.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+  </li>
+
+</ul>
+
+## Table
+
+The following is a basic markdown table.
+
+| Table | A column |
+| :--- | :--- |
```
Comment (polarweasel) [inline docs/content/preview/contribute/docs/all-page-elements.md:172]: Making the alignment markers line up with the column headings makes things easier.
```suggestion
| Table | A column |
| :---- | :------- |
```

File: docs/content/preview/contribute/docs/all-page-elements.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+
+{{< youtube id="qYMcNzWotkI" title="Deploy a fault tolerant cluster in YugabyteDB Managed" >}}
+
+## Lists
+
+What follows are some numbered steps, some code, text indented under a step:
+
+1. Numbered list with indented lists.
+
+    1. Second level numbered list indent.
```
Comment (polarweasel) [inline docs/content/preview/contribute/docs/all-page-elements.md:221]: Two-space indents are easier, instead of four...

**Comment by polarweasel** [comment]
I assume you're wanting to actually add this to the docs, not just use it for reference in the PR preview?

File: docs/content/preview/contribute/docs/all-page-elements.md (hunk 2 above)
Comment (ddhodge) [reply docs/content/preview/contribute/docs/all-page-elements.md:221]: Provided it renders correctly. In some situations I find 4 is needed for the indentation to work correctly...

**Comment by ddhodge** [comment]
Eventually. Also, is anything (important) missing.

File: docs/content/preview/contribute/docs/all-page-elements.md (hunk 2 above)
Comment (polarweasel) [reply docs/content/preview/contribute/docs/all-page-elements.md:221]: Four it is! And I just broke a list like this today, so your point is well taken. ;-)

**Pull request merged**